### PR TITLE
Chrimc/id

### PR DESCRIPTION
**Pull request by chrimc62**
$id was not a URI and was found in included schema.  This broke the dotnet schema validation.

### CODE REVIEW

File: packages/dialog/README.md [hunk 1]
```diff
@@ -26,89 +27,137 @@ OPTIONS
   -h, --help  show CLI help
 ```
 
-_See code: [src/commands/index.ts](https://github.com/microsoft/botframework-cli/src/commands/index.ts)_
+_See code: [src\commands\index.ts](https://github.com/microsoft/botframework-cli/blob/v1.0.0/src\commands\index.ts)_
```
Comment (cleemullins) [inline packages/dialog/README.md:30]: This looks odd - the mixture of "/" and "\\" probably isn't correct.

File: packages/dialog/README.md [hunk 2]
```diff
@@ -26,89 +27,137 @@ OPTIONS
...
+  $ bf dialog
 
 OPTIONS
-  -b, --branch=branch  [default: master] The branch to use for the meta-schema component.schema.
-  -h, --help           show CLI help
-  -o, --output=output  [default: app.schema] Output path and filename for merged schema. [default: app.schema]
+  -h, --help  Dialog command help
+```
+
+_See code: [src\commands\dialog\index.ts](https://github.com/microsoft/botframework-cli/blob/v1.0.0/src\commands\dialog\index.ts)_
```
Comment (cleemullins) [inline packages/dialog/README.md:44]: Same comment on URI slashes.

File: packages/dialog/README.md [hunk 3]
```diff
@@ -26,89 +27,137 @@ OPTIONS
...
+  -v, --verbose        Show verbose logging of files as they are processed.
 
-  --verbose            output verbose logging of files as they are processed
+EXAMPLES
+  $ bf dialog:merge *.csproj
+  $ bf dialog:merge libraries/**/*.schema **/*.csproj -o app.schema
 ```
 
-_See code: [src/commands/dialog/merge.ts](https://github.com/microsoft/botframework-cli/src/commands/dialog/merge.ts)_
+_See code: [src\commands\dialog\merge.ts](https://github.com/microsoft/botframework-cli/blob/v1.0.0/src\commands\dialog\merge.ts)_
```
Comment (cleemullins) [inline packages/dialog/README.md:67]: same.

File: packages/dialog/README.md [hunk 4]
```diff
@@ -26,89 +27,137 @@ OPTIONS
...
+ARGUMENTS
+  PATTERNS  Any number of glob regex patterns to match .dialog files.
 
 OPTIONS
   -h, --help  show CLI help
   --verbose   Show verbose output
 ```
 
-_See code: [src/commands/dialog/verify.ts](https://github.com/microsoft/botframework-cli/src/commands/dialog/verify.ts)_
+_See code: [src\commands\dialog\verify.ts](https://github.com/microsoft/botframework-cli/blob/v1.0.0/src\commands\dialog\verify.ts)_
```
Comment (cleemullins) [inline packages/dialog/README.md:85]: same

File: packages/dialog/README.md [hunk 5]
```diff
@@ -26,89 +27,137 @@ OPTIONS
...
+  -o, --output=output  [default: app.schema] Output path and filename for merged schema.
+  -v, --verbose        Show verbose logging of files as they are processed.
+
+EXAMPLES
+  $ bf dialog:merge *.csproj
+  $ bf dialog:merge libraries/**/*.schema **/*.csproj -o app.schema
 ```
 
-_See code: [src/commands/dialog/merge.ts](https://github.com/microsoft/botframework-cli/blob/v0.0.0/src/commands/dialog/merge.ts)_
+_See code: [src\commands\dialog\merge.ts](https://github.com/microsoft/botframework-cli/blob/v1.0.0/src\commands\dialog\merge.ts)_
```
Comment (cleemullins) [inline packages/dialog/README.md:143]: same

File: packages/dialog/src/commands/dialog/readme.md [hunk 6]
```diff
@@ -2,20 +2,16 @@
...
...chema JSON schema files into a single JSON schema file. You can point to the files either directly with a glob pattern or indirectly through a glob pattern that matches a package.json, packages.config or \*.csproj file. The .schema files should have a unique filename that is used to refer to that type using `$kind`. The .schema files should include a `$schema: "https://raw.githubusercontent.com/microsoft/botframework-sdk/master/schemas/component/component.schema"` which defines the schema they are validated against. You can access common definitions by using `$ref: "schema:#/definitions/stringExpression"` in a property definition.  At the top-level in a component schema you can use `$role:"implements(<kind>)` to project the type definition into interface types defined using `$role:"interface"` while merging. To extend an existing .schema file use `$role: "extends(<kind>)"`.  To refer to a type in a property, use `"$kind":"<kind>"` and the corresponding kind will be wired into your schema.  The merger combines all of the component .schema files into a single .schema file that has resolved all external `$ref`, merged `allOf` and connected together schemas through `$role` and `$kind`.
```
Comment (cleemullins) [inline packages/dialog/src/commands/dialog/readme.md:5]: grammer / spelling. "ths" vs This

File: packages/dialog/README.md (hunk 1 above)
Comment (chrimc62) [reply packages/dialog/README.md:30]: This is automatically generated.

File: packages/dialog/README.md (hunk 2 above)
Comment (chrimc62) [reply packages/dialog/README.md:44]: Same here.

File: packages/dialog/README.md (hunk 1 above)
Comment (chrimc62) [reply packages/dialog/README.md:30]: Automatically generated.

**Pull request merged**